scripts/qc_report_per_sample.rmd

```diff
@@ -20,11 +20,18 @@ library(stringr)
 library(magrittr)
 library(openssl)
 ```
+
 # Input Settings
 ```{r InputSettings, echo = FALSE, include = FALSE}
 coverage_file <- params$coverage_file
 sample_name <- params$sample_name
 outFile <- params$outFile
+fastqc_dir <- params$fastqc_dir # TODO @Jan @Miri add that argument to the snakemake rule and the rendersite function?
+# TODO: @Jan snakemake needs to be run twice before and after the preprocessing
+# TODO: @Jan @Ricardo 
+fastqc_unprocessed <- "FASTQ_DIR/{sample}_unprocessed_fastqc.html"
+fastqc_processed <- "FASTQ_DIR/{sample}_unprocessed_fastqc.html"
```

Comment:
should be "_processed_fastqc.html", no?

Reply:
FASTQ_DIR is a string, but I guess you want it to be a variable, such as the earlier `fastqc_dir`, no?

Reply:
fixed